### PR TITLE
Build - 1

### PR DESCRIPTION
**Pull request by akoushke**

### CODE REVIEW

File: .circleci/config.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          paths: dist
+
+# execute the jobs in a orderly manner
+workflows:
+  setup_test_release:
+    jobs:
+      - install
+      - build:
+          requires:
+            - test
```
Comment (adamweeks) [inline .circleci/config.xml]: Requires test? I don't see that defined

Comment (akoushke) [reply .circleci/config.xml]: why is this not running it on the CI???

Comment (akoushke) [reply .circleci/config.xml]: okay addressed

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      - persist_to_workspace:
+          root: .
+          paths: dist
+
+# execute the jobs in a orderly manner
+workflows:
+  setup_test_release:
+    jobs:
+      - install
+      - build:
```
Comment (lalli-flores) [inline .circleci/config.yml]: Isn't this only supposed to run on `master`?

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+          command: npm run build
+      - persist_to_workspace:
+          root: .
+          paths: dist
+
+# execute the jobs in a orderly manner
+workflows:
+  setup_test_release:
+    jobs:
+      - install
```
Comment (lalli-flores) [inline .circleci/config.yml]: Where's the test step?

File: .circleci/config.yml (hunk 2 above)
Comment (akoushke) [reply .circleci/config.yml]: yeah, but I wanted to run it to test.

File: .circleci/config.yml (hunk 3 above)
Comment (akoushke) [reply .circleci/config.yml]: will be added in JEST PR

**Pull request merged**